src/control/cmd/daos/pretty/health.go

```diff
@@ -0,0 +1,174 @@
+//
+// (C) Copyright 2024 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package pretty
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/dustin/go-humanize"
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/build"
+	"github.com/daos-stack/daos/src/control/lib/daos"
+	"github.com/daos-stack/daos/src/control/lib/txtfmt"
+)
+
+const (
+	minDisplayImbalance = 10
+)
+
+func printBuildInfo(out io.Writer, comp build.Component, cb *daos.ComponentBuild) {
+	if cb == nil {
+		return
+	}
+
+	buildInfo := cb.BuildInfo
+	if buildInfo == "" {
+		buildInfo = cb.Version
+	}
+	fmt.Fprintf(out, "%s: %s\n", txtfmt.Title(comp.String()), buildInfo)
+}
+
+func printSystemInfo(out io.Writer, si *daos.SystemInfo, verbose bool) {
+	if si == nil {
+		return
+	}
+
+	fmt.Fprintf(out, "System Name: %s\n", si.Name)
+	fmt.Fprintf(out, "Fabric Provider: %s\n", si.Provider)
+	apStr := "[]"
+	if len(si.RankURIs) > 0 {
+		if apBytes, err := json.Marshal(si.AccessPoints()); err == nil {
+			apStr = string(apBytes)
```

Comment:
Marshaling JSON here seems a bit odd to me.

Reply:
It's just a quick way to stringify the AP list, with each entry quoted. The output could be copy/pasted right into a config file. The alternative is building a new slice and iterating through it to quote each entry, then joining the new slice and manually enclosing it in brackets.

Reply:
That's fair. I hadn't thought of the config file use case (although wouldn't we then want to marshal it to yaml?).

Reply:
Technically, [YAML is a superset of JSON](https://en.wikipedia.org/wiki/JSON#YAML), so the YAML parser will accept JSON. 😄

Reply:
Wow. That's good info @mjmac :)